# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/editing/run/undo-redo-after-mutation.html.ini
# Commit: 06bcc4f14cfa
# Full Hash: 06bcc4f14cfab44c0b8435aaffabd5d3736ba379
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2020-05-07 16:27:48
# Description:
#   Bug 1633700 - Make `CreateElementTransaction::InsertNewNode()` check the relation between container and child only when the point has child node r=m_kato
#   
#   I guess that this crash may occur in these conditions:
#   
#   1. It pointed end of a node, but the node has fewer children at redo.
# ==============================================================================

diff -r c3c27cb46db6 -r 06bcc4f14cfa testing/web-platform/meta/editing/run/undo-redo-after-mutation.html.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/meta/editing/run/undo-redo-after-mutation.html.ini	Thu May 07 02:25:03 2020 +0000
@@ -0,0 +1,3 @@
+[undo-redo-after-mutation.html]
+  [Redo for execCommand("insertLineBreak") between <b> and <i> after after removing following <i> element with DOM API after undoing]
+    expected: FAIL